spec/factories/gias_schools.rb

```diff
@@ -8,6 +8,7 @@
     town { Faker::Address.city }
     postcode { Faker::Address.postcode }
     status_code { GiasSchool::ESTABLISHMENT_OPEN_STATUS_CODE }
+    searchable { %w[name urn postcode] }
```

Comment:
Do we need to set this, the callback should generate one when it's created?

Reply:
Good point